zipkin-server/src/main/java/zipkin2/server/internal/ZipkinServerConfiguration.java

```diff
@@ -68,6 +68,10 @@
   @Autowired(required = false)
   ZipkinMetricsHealthController healthController;
 
+  @Bean @ConditionalOnMissingBean ObjectMapper objectMapper() {
+    return new ObjectMapper();
```

Comment:
Does the current server use this or is it just for safety? Wondering if the default object mapper should ignore missing properties (good for JSON APIs where fields get added or ignored, which might be true for our ES client).

Reply:
`ZipkinMetricsHealthController` uses this, though databind is indeed used primarily by ES. I'll scrub it out as better to be aware of where we are using databind (one of our largest dependencies)

Reply:
(scrub it from the server, to avoid this bean, not from ES at the moment)

Reply:
https://github.com/openzipkin/zipkin/pull/2819 fixes this

Reply:
nice catch btw